CHANGELOG.md

```diff
@@ -3,3 +3,6 @@
 
 ## 1.0.1
   - internal: rename DeadLetterQueueManager to DeadLetterQueueReader
+
+## 1.0.2
```

Comment:
we usually place the versions in the reverse order

example: https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/master/CHANGELOG.md